shared/fixes/bash/accounts_authorized_local_users_sidadm_orasid.sh

```diff
@@ -0,0 +1,56 @@
+# platform = multi_platform_ol
+. /usr/share/scap-security-guide/remediation_functions
+populate var_accounts_authorized_local_users_regex
+
+# never delete the root user
+default_os_user="root"
+
+# add users sidamd and orasid
+userlist="root"
+
+# if /sapmnt/SID exists, add sidadm to the list
+SIDlist=$(find /sapmnt -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]$')
+for i in $SIDlist ; do
+	SID=${i:8:3}
+	userlist="$userlist|$(echo "$SID" | sed -e 's/\(.*\)/\L\1/')adm"
+done
+
+# if /sapmnt/SID/exe/brspace or /sapmnt/SID/exe/<type>/<platform>/brspace exists,
+# add orasid to the list
+SIDlist=$(find /sapmnt -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]/exe/brspace$' \
+	-o -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]/exe/uc/[a-z0-9_]+/brspace$' \
+	-o -regex '^/sapmnt/[A-Z][A-Z0-9][A-Z0-9]/exe/nuc/[a-z0-9_]+/brspace$')
+for i in $SIDlist ; do
+        SID=${i:8:3}
```

Comment:
Please try to change naming here - if I have a SIDlist, then I expect one element of it to be SID. However, SID is just part of element of SIDlist, which is somehow confusing.